ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-create-test.js

```diff
@@ -1008,40 +1014,6 @@ module('Acceptance | kv-v2 workflow | secret and version create', function (hook
       assert.strictEqual(currentURL(), `/vault/secrets/${backend}/kv/list/app/`, 'sub-dir page');
       assert.dom(PAGE.list.item()).doesNotExist('Does not list any secrets');
     });
-    test('create new version of secret from older version (sc)', async function (assert) {
```

Comment:
I deleted this test because this user's policy is:
```
path "secret/data/*" {
   capabilities = ["create", "update"]
}
path "secret/metadata/*" {
   capabilities = ["delete"]
}
```
with no read capabilities users cannot navigate to older versions of a secrert. if a user were to type in the version in the URL then this would result in a save error unless they happened to get the `cas` version right. 

instead I moved the warning assertion up to the create new version test above.